### ISSUE TITLE
is this project still on?

### ISSUE THREAD

**Issue by XobilePidor** [Lockzi/macOS-XPS-9700-Catalina#1]
been dying for an XPS 9700 hackintosh, is this project still on? any progress mate?

**Comment by laris** [comment]
P5750, also struggling 
Big Sur?

**Comment by VGerris** [comment]
and does sound work?

**Comment by laris** [comment]
yeah, support Sonoma, telegram have a chatroom and i share the config.plist there.

**Comment by VGerris** [comment]
> yeah, support Sonoma, telegram have a chatroom and i share the config.plist there.

Hi, the repo shows 3 years old and there is barely information. Can you update it with the Sonoma version so we can test sound and sleep? I haven't found any working EFI for that, I am sure many would be greatful! Thank you
